src/land/ascat_ssm2ioda.py

```diff
@@ -107,7 +111,8 @@ def _read(self):
         for i in range(len(lons)):
             base_date = datetime(2000, 1, 1) + timedelta(seconds=int(secs[i]))
             base_datetime = base_date.strftime("%Y-%m-%dT%H:%M:%SZ")
-            AttrData['date_time_string'] = base_datetime
+            # date_time_string is deprecated  (2022/03/09)
+#           AttrData['date_time_string'] = base_datetime
```

Comment:
Remove these two lines

Reply:
This has been resolved by @gthompsnJCSDA. Please, see the commits below.